changelog/unreleased/kong-manager/a11y-improvements.yml

```diff
@@ -0,0 +1,2 @@
+message: Improved accessibility in Kong Manager.
```

Comment:
This seems very general. Do we need it?

Reply:
Perhaps, It would be better to provide more details.

Reply:
It's not a problem to have general changelog entries. This one specifically comprises improvements in several aspects: adding `aria-*` labels to indicate different statuses of an element, supporting keyboard navigation and events in more components, etc. Listing them all will make it too long so I think it's acceptable to leave it as is.